web/src/OverviewResourcePane.tsx

```diff
@@ -88,13 +97,15 @@ export default function OverviewResourcePane(props: OverviewResourcePaneProps) {
         {...starredResourcePropsFromView(props.view, selectedTab)}
       />
       <Main>
-        <OverviewResourceSidebar {...props} name={name} />
-        <OverviewResourceDetails
-          resource={r}
-          name={name}
-          alerts={alerts}
-          buttons={buttons}
-        />
+        <SplitPane split="vertical" minSize={336} defaultSize={336}>
```

Comment:
what do you think about adding the split pane to the storybook component for `OverviewResourceSidebar`? it might be nice to have that as part of the demo view.

Reply:
ooh i like that idea!